setup/fio-setup-basic

```diff
@@ -180,6 +203,32 @@ __numa_cpu_nodes()
 parse_numa_mem_policy
 parse_numa_cpu_nodes
 
+parse_cpus_allowed_policy()
+{
+        if [ "$cpus_allowed_policy" = "split" ]; then
+                cpus_allowed=$cpus_allowed
+                echo "cpus_allowed_policy=${cpus_allowed_policy}"
+        elif [ "$cpus_allowed_policy" = "shared" ]; then
+                cpus_allowed=$cpus_allowed
+                echo "cpus_allowed_policy=${cpus_allowed_policy}"
+        fi
+
+}
+
+parse_cpus_allowed()
+{
+        if [ "$cpus_allowed_policy" = "split" ]; then
+                [ -n "$cpus_allowed" ] || die "cpus_allowed must be specified for fio"
+                cpus_allowed=$cpus_allowed
```

Comment:
how about this line, what's the meaning?

Reply:
can refer to "man fio"
![image](https://github.com/intel/lkp-tests/assets/166705036/8bae513b-66ff-4ca1-aa23-f47e5946799e)
if  $cpus_allowed_policy is set to split, it means, it need to bind cpu cores.

if "$cpus_allowed" is set and not null  it is True.

otherwise,

if not True ("$cpus_allowed") is not set or it is null, response error "cpus_allowed must be specified for fio"

Reply:
Similar implementation, can refer to https://github.com/intel/lkp-tests/blob/master/setup/fio-setup-basic#L33